src/Elastic.Apm/Report/Serialization/TrimmedStringJsonConverter.cs

```diff
@@ -1,12 +1,19 @@
 using System;
+using System.Runtime.CompilerServices;
 using Newtonsoft.Json;
 
 namespace Elastic.Apm.Report.Serialization
 {
 	internal class TrimmedStringJsonConverter : JsonConverter<string>
 	{
+		private readonly int _maxLength;
+
+		public TrimmedStringJsonConverter() : this(Consts.PropertyMaxLength) { }
```

Comment:
Just to make resharper and rider happy, could you please add `// ReSharper disable once UnusedMember.Global` before this one?

Reply:
fixed